crates/bevy_winit/src/lib.rs

```diff
@@ -370,6 +400,7 @@ pub fn winit_runner(mut app: App) {
                 );
                 app.update();
             }
+
```

Comment:
Other matches are not separated.

Reply:
I'm not sure what you're saying here sorry, could you explain please?